portal/_header.php

```diff
@@ -29,24 +32,21 @@
 <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
 <meta name="description" content="Developed By sjpadgett@gmail.com">
 
-<link href="<?php echo $GLOBALS['assets_static_relative']; ?>/font-awesome-4-6-3/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
-
-<link href="<?php echo $GLOBALS['assets_static_relative']; ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
-<?php if ($_SESSION['language_direction'] == 'rtl') { ?>
-    <link href="<?php echo $GLOBALS['assets_static_relative']; ?>/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css" rel="stylesheet" type="text/css" />
-<?php } ?>
-
-
+    <?php $css = $GLOBALS['css_header'];
+    $GLOBALS['css_header'] = "";
+    Header::setupHeader(['datetime-picker', 'jquery-ui', 'jquery-ui-sunny', 'emodal']);
```

Comment:
Do we need a feature in this function to skip brining in css_header?
(I can likely add this, if needed)

Reply:
I actually brought this up when I was pushing to get the dependency order changed to also allow excluding style sheets but, the only admin to comment on any of it was you. I can think of occasions you may not want or need to include styles but want the core dependencies. So this is my work around.

Reply:
Just checked in your belated Christmas gift into the codebase :)
https://github.com/openemr/openemr/commit/efc948f9b81ee9b3fc4c1e9add8b09f1f8e0abb7
So, can do this now instead for above:
``````
Header::setupHeader(['no_main-theme', 'datetime-picker', 'jquery-ui', 'jquery-ui-sunny', 'emodal']);
``````